navigation-testing/src/androidMain/kotlin/com/freeletics/khonshu/navigation/NavigatorTurbine.kt

```diff
@@ -188,7 +187,7 @@ public interface NavigatorTurbine {
      *
```

Comment:
nit: description in ln 185 needs to be adapted too

Reply:
https://github.com/freeletics/khonshu/pull/922/commits/4b3383c5db9595f230ae8781bf2a71f2cefa2001